docs/reference/contrib/settings.rst

```diff
@@ -108,17 +108,20 @@ Settings are designed to be used both in Python code, and in templates.
 Using in Python
 ---------------
 
-If access to a setting is required in the code, the :func:`~wagtail.contrib.settings.models.BaseSetting.for_site` method will retrieve the setting for the supplied site:
+If you require access to a setting in a view, the :func:`~wagtail.contrib.settings.models.BaseSetting.for_request` allows you to retrieve the relevant settings for the current request:
```

Comment:
Missing "method" before "allows you to...". Or, can be viewed as having an extra "the" in front of `:func:...`